packages/web/src/pages/app/contributions.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { NextPage } from 'next';
+import { Heading, SimpleGrid, Text, VStack } from '@chakra-ui/react';
+import { AppLayout } from '../../components/Layout';
+import { ContributionsBox } from '../../components/Contributions';
+
+export interface ContributionsList {
+  id: string;
+  nodeID: string;
+  type: string;
+  score: number;
+  contributedAt: Date;
+  description: string | null;
+  url: string;
+}
+
+const Contributions: NextPage = () => {
+  const [Contributions, setContributions] = React.useState<ContributionsList[]>([]);
+
+  React.useEffect(() => {
+    const fetchContributions = async () => {
+      // Get all Contributions
+      const res = await fetch('/api/contributions');
+      const ContributionsList = await res.json();
+
+      // Set contribution list
+      setContributions(ContributionsList);
+    };
+
+    fetchContributions();
+  }, []);
+
+  return (
+    <AppLayout>
+      <VStack spacing={3}>
+        <Heading>Contributions</Heading>
+      </VStack>
+      {Contributions.length <= 0 ? (
```

Comment:
```suggestion
      {contributions.length <= 0 ? (
```

Reply:
You'll need to flip these, but it's simpler to just check the truthiness of `contributions`:
```suggestion
      {Contributions.length ? (
```